tests/e2e/machineregistration_test.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_test
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	http "github.com/rancher-sandbox/ele-testhelpers/http"
+	kubectl "github.com/rancher-sandbox/ele-testhelpers/kubectl"
+
+	"github.com/rancher-sandbox/rancheros-operator/tests/catalog"
+)
+
+var _ = Describe("MachineRegistration e2e tests", func() {
+	k := kubectl.New()
+	Context("registration", func() {
```

Comment:
Is `Context` really useful here? As there is only this one (or maybe more are planned?).

Reply:
I hoped there was more in the pipeline for that change at hand :sweat_smile: but this will get bigger I'm pretty sure..